docs/conf.py

```diff
@@ -92,4 +92,5 @@
     'requests': ('https://2.python-requests.org/en/stable/', None),
```

Comment:
```suggestion
     'requests': ('https://requests.readthedocs.io/en/stable/', None),
```
Just found a fresh URL for `requests`